wcfsetup/install/files/lib/data/user/UserAction.class.php

```diff
@@ -618,6 +618,62 @@ public function validateDisable() {
 		$this->validateEnable();
 	}
 	
+	/**
+	 * Validates the confirm email action.
+	 * @since       5.3
+	 */
+	public function validateConfirmEmail() {
+		$this->validateEnable();
+	}
+	
+	/**
+	 * Validates the unconfirm email action.
+	 * @since       5.3
+	 */
+	public function validateUnconfirmEmail() {
+		$this->validateEnable();
+	}
+	
+	/**
+	 * Marks the email address as confirmed. It also enables the user, iff the register method is user activation only.
```

Comment:
`iff`

Reply:
https://en.wiktionary.org/wiki/iff

Reply:
Interesting, I didn't know that term until now. Thanks! 😛